frontend/src/metabase/query_builder/components/chart-type-selector/ChartTypeList/ChartTypeList.tsx

```diff
@@ -0,0 +1,29 @@
+import { ChartTypeOption, type ChartTypeOptionProps } from "../ChartTypeOption";
+
+import { OptionList } from "./ChartTypeList.styled";
+
+export type ChartTypeListProps = {
+  visualizationList: ChartTypeOptionProps["visualizationType"][];
```

Comment:
yeah, I was kinda on the fence about this one. My thinking is that we should couple `ChartTypeOptionProps` with `ChartTypeListProps` since the components are linked, so if we ever decide to change the input of `ChartTypeOption` then it's built in